CHANGELOG.asciidoc

```diff
@@ -57,6 +57,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Fix bug in loading dashboards from zip file. {issue}8051[8051]
 - The export config subcommand should not display real value for field reference. {pull}xxx[xxx]
 - The export config subcommand should not display real value for field reference. {pull}8769[8769]
+- Update Beats to use go 1.11.2 {pull}8746[8746]
```

Comment:
This changelog is mostly for users, who do not care about compiler versions :)

Let's add the entry to the developers changelog.

Reply:
@vjsamuel Can you clean up the changelog?

Reply:
I don't think we need it in the CHANGELOG.asciidoc. The developers changelog should be good enough